pkg/services/searchV2/bluge.go

```diff
@@ -531,7 +537,7 @@ func doSearchQuery(ctx context.Context, logger log.Logger, reader *bluge.Reader,
 			case documentFieldTag:
 				tags = append(tags, string(value))
 			default:
-				return ext(field, value)
+				ext(field, value)
```

Comment:
returning anything except "true" will break things, so lets avoid that chance from the enterprise extension :)